src/routes/blog/index.json.ts

```diff
@@ -1,9 +1,22 @@
+import { readFileSync } from 'fs';
 import type { RequestHandlerOutput } from '@sveltejs/kit/types/internal';
 import { fetchNewsletters } from './_query';
+import { DEBUG } from '$lib/constants';
+
+const SAMPLE_POST = readFileSync('./src/routes/blog/_testdata/posts.json');
```

Comment:
Nit: This variable represents a list of blog posts. 
```suggestion
const SAMPLE_POSTS = readFileSync('./src/routes/blog/_testdata/posts.json');
```

Reply:
Done.